modules/redis-cluster/outputs.tf

```diff
@@ -38,3 +38,12 @@ output "redis_cluster" {
   description = "The redis cluster created"
   value       = google_redis_cluster.redis_cluster
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//redis.googleapis.com/${google_redis_cluster.redis_cluster.id}"
```

Comment:
This is CAIS style service uri needed by AppHub. I see it is same as Redis Instance (`//redis.googleapis.com`). Is that the case?